contracts/FROST.sol

```diff
@@ -279,7 +294,15 @@ library FROST {
             }
         }
 
+        // e must be in range [1, _N) because of ecrecover requirements
+        // e in range [0, _N) because of _hashToField
         uint256 e = _challenge(rx, ry, px, py, message);
+        //bool eOk = _isScalar(e);
+        bool eOk = e != 0;
+        if (!eOk) {
+            return address(0);
+        }
+
```

Comment:
I don't think this is necessary - this would mean you have a pre-image for the `sha256(...) == 0` which is computationally impossible.